frontend/constants/measurements.ts

```diff
@@ -7,6 +7,7 @@ const MEASUREMENTS = {
     HEADER_SHADOW: 'default',
     TITLE_CLASS: '1.5rem',
     TITLE_MARGIN: '15px',
+    TITLE_SPACING: '0rem',
```

Comment:
Using the `inherit` keyword might be better in this case, not sure. I assume the default is something like `0rem` anyways.